install/installation/installation.go

```diff
@@ -310,46 +322,53 @@ func (k KymaInstaller) installTiller(tillerYaml string, createFunction func([]k8
 	return nil
 }
 
-func (k KymaInstaller) deployInstallerForIstallation(installerYaml string) error {
-	return k.deployInstaller(installerYaml, k.k8sGenericClient.CreateResources)
+func (k KymaInstaller) deployInstallerForIstallation(yamlFile string) error {
+	return k.deployInstaller(yamlFile, k.k8sGenericClient.CreateResources)
 }
 
-func (k KymaInstaller) deployInstallerForUpgrade(installerYaml string) error {
-	return k.deployInstaller(installerYaml, k.k8sGenericClient.ApplyResources)
+func (k KymaInstaller) deployInstallerForUpgrade(yamlFile string) error {
+	return k.deployInstaller(yamlFile, k.k8sGenericClient.ApplyResources)
 }
 
-func (k KymaInstaller) deployInstaller(installerYaml string, createResourcesFunc func(resources []k8s.K8sObject) ([]*unstructured.Unstructured, error)) error {
+func (k KymaInstaller) deployInstaller(yamlFile string, createResourcesFunc func(resources []k8s.K8sObject) ([]*unstructured.Unstructured, error)) error {
 	k.infof("Deploying Installer...")
 
-	k8sInstallerObjects, err := k8s.ParseYamlToK8sObjects(k.decoder, installerYaml)
+	k8sInstallerObjects, err := k8s.ParseYamlToK8sObjects(k.decoder, yamlFile)
 	if err != nil {
-		return fmt.Errorf("failed to parse Installer yaml file to Kubernetes dynamicClientObjects: %w", err)
+		return fmt.Errorf("failed to parse yaml file to Kubernetes dynamicClientObjects: %w", err)
 	}
 
-	var installationCR *v1alpha1.Installation
-	installationCR, k8sInstallerObjects, err = k.extractInstallationCR(k8sInstallerObjects)
```

Comment:
We do extract the Installation CR from the artifacts here so I do not understand what this change actually achieves.